backend/routes/Event.js

```diff
@@ -1,20 +1,20 @@
 // Create Event routers
-const express = require("express");
+const express = require('express');
 const {
   createEvent,
   getAllEvents,
   modifyEvent,
   finishEvent,
-} = require("../controllers/Event");
-const { protect } = require("../utils/auth");
-const { inviteUsers } = require("../controllers/EventParticipation");
+} = require('../controllers/Event');
+const { protect } = require('../utils/auth');
+const { inviteUsers } = require('../controllers/EventParticipation');
 const EventRouter = express.Router();
 
 // Set up routes
-EventRouter.post("/create", protect, createEvent);
-EventRouter.post("/invite/:id", protect, inviteUsers);
-EventRouter.get("/get", protect, getAllEvents);
-EventRouter.put("/modify", protect, modifyEvent);
-EventRouter.post("/finish", protect, finishEvent);
+EventRouter.post('/create', protect, createEvent);
+EventRouter.post('/invite/:id', protect, inviteUsers);
```

Comment:
Add validations as middleware as well. Currently you only check if user is authenticated. What if event is missing a name? What if invite is missing a list of emails?